### PR TITLE
Add mini-app example

### PR DESCRIPTION
**Pull request by borja-munoz**
Add the mini-app example to the examples/_debug folder

### CODE REVIEW

**Comment by VictorVelarde** [comment]
Vercel is available now: https://web-sdk-git-feature-include-miniapp-example.carto-frontend.now.sh/examples/_debug/mini-app/mini-app.html

I have seen some small changes we need to make before merging, as right now we have some issues (event renames...). I will review it during the next week, after the release

File: examples/_debug/mini-app/mini-app.html [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+            // Pop-ups can be added before or after the layer has been added to the map
+            await worldCommerceLayer.setPopupHover([
+                {
+                    attr: 'country',
+                    title: ''                       // How to hide the field name
+                },
+                {
+                    attr: 'gdp',
+                    title: 'GDP',
+                    format: '.4s'
```
Comment (manmorjim) [inline examples/_debug/mini-app/mini-app.html]: this D3 format is not available anymore

**Review by manmorjim — CHANGES_REQUESTED** [review]

File: examples/_debug/mini-app/mini-app.html (hunk 1 above)
Comment (borja-munoz) [reply examples/_debug/mini-app/mini-app.html]: Fixed

**Review by VictorVelarde — APPROVED** [review]
LGTM.

**Pull request merged**